utils/githooks/prepare-commit-msg

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -eux
+
+run-parts() {
+    local dir="$1"
+    shift
+
+    for i in $(LC_ALL=C; echo "${dir%/}"/*[^~,]); do
+        # don't run vim .swp files
+        [ "${i%.sw?}" != "${i}" ] && continue
+        $i "$@"
+    done
+}
```

Comment:
Seeing this function copy-pasted makes me think we need a `run_parts.sh`. Especially if we need to ignore more than just `.sw?`